test/tools/mongodb-mock/index.js

```diff
@@ -91,7 +92,7 @@ function cleanup(spy, callback) {
 module.exports = {
   createServer,
   cleanup,
-  DEFAULT_ISMASTER,
```

Comment:
@nbbeeken Since this code is new to this location, I'm wondering if we get this PR merged before the release if we can rename these without deprecation? From what I can tell, our code is not using DEFAULT_LEGACY_HELLO and  DEFAULT_LEGACY_HELLO_36, so I'm wondering if those should be deleted?

Reply:
I removed the constants and the tests are passing.